.travis.yml

```diff
@@ -46,13 +23,9 @@ matrix:
       env: TOXENV=style
     - python: 3.7
       env: TOXENV=readme
-  allow_failures:
-    - env: TOXENV=py35-djmaster
-    - env: TOXENV=py36-djmaster
-    - env: TOXENV=py37-djmaster
 
 install:
-  - pip install tox codecov
+  - pip install tox tox-travis codecov
```

Comment:
Please don't. tox-travis hides jobs for individual Django versions in the Travis CI build overview.